### DISCUSSION TITLE
整合包2.1.2版本中让物品需要手动捡起的模组是哪一个？

### DISCUSSION THREAD

**Discussion by Silvigarabis** [Rad586/Forget-Me-Not#80] [Q&A] [answered]
整合包2.1.2版本中让物品需要手动捡起的模组是哪一个？或者说有关的配置项是哪一条？

我最近通过转移存档的方式，把整合包从2.1.2版本升级到了2.8.0，但是我发现东西不能通过手动的方式拾取了，而是隔一段时间自动拾取。这让我不太习惯，所以想改回去，但是……没找到配置在哪里

**Comment by Rad586** [comment, marked as answer]
原本的模组是interactic。
在新版本，手动拾取的模组已经被替换成了“toggled pickup”，你可以按`~`键切换手动（shift）和自动拾取（原版）。
2.5.0+, Interactic has been replaced by Toggled Pickup, you can use `~` on your keyboard to toggle manually(press `shift`) or auto(vanilla behavior) pickup.
